### ISSUE TITLE
Not an issue, a question: mulitple todo.txt files open at once?

### ISSUE THREAD

**Issue by ultrablue** [dbeniamine/todo.txt-vim#1]
Hi:

Is it possible to have multiple todo.txt files open and have the excellent goodness of your (and freitass') code work with them, too?

Thanks!
Greg

**Comment by dbeniamine** [comment]
Hello,

I'm not sur I understand well your question, but if you open several todo.txt
files, everything is going to work well and completely independently in each
file. Still I agree that completion should provide hints from all open
todo.txt files. If you are interested by this functionality, I can add it to
my todo.txt list and implement it ASAP.

Best regards,
Bye,

David

**Comment by ultrablue** [comment]
Hi, David:

I'm not sure that I've grabbed your latest commit correctly, but I have been messing around. So it looks like as long as I name a file todo.txt, your code realized that it's supposed to do it's thing? So now I've got this:
today/todo.txt.
However, when I try to mark a task as done (<LocalLeader>x) on that buffer, I get this error:
Error detected while processing function TodoTxtToggleMarkAsDone..TodoTxtMarkAsDone:
line    3:
E21: Cannot make changes, 'modifiable' is off

And

**warning** (netrw) using Pexplore or <s-up> improperly; see help for netrw-starstar

So maybe I didn't get the correct code?

Sorry to be dense -- I'm a bit of noob with both vim and github. :/

Thanks,
Greg

**Comment by dbeniamine** [comment]
Hello Greg,

Let's answer point by point:

1. Todo.txt-vim detect as a todo file all files matching one of the following
    + [Tt]odo.txt
    + [Dd]one.txt
2. I don't think the problem comes from todo.txt-vim, according to the error
message, the plugin is not even able to edit the buffer (not saving, just
editing). Can you modify it manually ?
3. The error message seems to come from vim netwr, are you editing a file over
a network ? Can you explain more precisely where is the file, how are you
doing things ?

Bests,

David

**Comment by ultrablue** [comment]
Hi, David:

Sorry for the delay and thank you so much for your response.

tl;dr: uninstall Freitass and re-install dbeniamine.

1. Understood.
2. Yup. I'm able to edit and save my todo.txt files.
3. Hm. The file is local to my machine; no network. So here are the details:
OS X 10.9.5
MacVim version 7.4.258
Open a todo.txt file (by that I mean file who's name is todo.txt).
Note that it appears to be coloring (A), (B), and (C) tasks correctly.
Goto any line. Type <localleader>x.
I get the errors above.

Here's something though: I didn't remove the Freitass version before installing yours. Let me try that...
Ok, I went in and removed everything that was in the Fretass version from ~/.vim.
Copied your code as described in the Readme.
Restarted MacVim.
Now, <localleader>x has the desired result: marking a task complete.
Let me try opening another, different todo.txt file...
And that's working as expected, as well.
I'm now seeing some other issues which could be bugs or they could be more along these lines.

I'll report them as I see them.

Thanks for your help!
Greg

**Comment by dbeniamine** [comment]
If I understand well, this problem is solved so I close the issue.

If you encounter some other problems feel free to report them.

Bests,
David
